http_check/tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import pytest
+from mock import patch
+from common import HERE
+
+
+@pytest.fixture
+def aggregator():
+    from datadog_checks.stubs import aggregator
+    aggregator.reset()
+    return aggregator
+
+
+@pytest.fixture(scope='session')
+def http_check():
+    from datadog_checks.http_check import HTTPCheck
+    # Patch the function to return the certs located in the `tests/` folder
+    patcher = patch('datadog_checks.http_check.http_check.get_ca_certs_path', new=mock_get_ca_certs_path)
+    patcher.start()
+    http_check = HTTPCheck('http_check', {}, {})
+    yield http_check
+    patcher.stop()
+
+
+def mock_get_ca_certs_path():
+    """
+    Mimic get_ca_certs_path(), but use the certificates located in the `tests/` folder
+    """
+
+    # the embedded_root is mimiced in this (tests/) folder
+    embedded_certs = os.path.join(HERE, 'fixtures', 'cacert.pem')
```

Comment:
I copied the certs included in the Agent to the `tests/` folder so devs can test the integration without requiring the Agent to be installed.

As well, this avoids the issue of actually finding the certs themselves.